src/BaseTemplates/WebAPI/web.config

```diff
@@ -9,6 +9,6 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified"/>
     </handlers>
-    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" startupTimeLimit="3600" stdoutLogEnabled="false"/>
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" stdoutLogEnabled="false" stdoutLogFile=".\logs\stdout" forwardWindowsAuthToken="false"/>
```

Comment:
Does forwardWindowsAuthToken need to be set to true for the Windows Auth template? If so we need to lay down a different web.config for that template